sass/components/Dashboard/utilization.scss

```diff
@@ -35,3 +35,7 @@
   border-left-style: solid;
   border-left-color: rgb(209, 209, 209);
 }
+
+.kubevirt-utilization__card {
```

Comment:
I don't think this is gonna work on all devices. + this does not belong to `Dashboard/`